### ISSUE TITLE
Create GitHub action to run the script to probe the Service control policies (SCP)

### ISSUE THREAD

**Issue by crochcunill** [bcgov/cloud-pathfinder#2251]
**Describe the Issue**
As a tester I want be able to run the scripts automatically from a  GitHub action, if possible using OIDC
(note: target accounts for this test have to be OIDC enabled)

There are two sets of scripts that can be added to a GitHub CI/CD using OIDC authentication. They are the scripts that probe the Service Control Policies (**aws-ecf-probing-scp**  repo) and  the scripts that gather information about the current state of the Landing Zone(s) (**aws-sea-guardrail-scanner** repo)

Both scripts are currently executed from the testers workstation command line. This is fine, however, in the context of automating all the procedures as much as possible, it would be better these scripts can be run by a GitHub action. Once this step is done, then can be easily integrated into a larger suite of tests that can be triggered by an event like a PR, effectively integrating them into a CI/CD pipeline.

Note: This task applies only to **aws-ecf-probing-scp** . Task #2258 is for  **aws-sea-guardrail-scanner** repo.

Note: Effort increased to 5. The reason is that now this ticket can be implemented using OIDC authentication, instead of using AWS permanent credentials. However, it means that it will be necessary to craft the appropriate policies, that is a time consuming taks

**Additional Context**
To run this test the tester is missing the required Live Master Landing Zone credentials. Creating a well crafted role for the master account to be used by GitHub action through OIDC may be a good solution that will free some cycles to SysOps by not having to run the test in Live LZ

A possible caveat is that each time a new SCP is probed, the policies associated to the role will need to be edited. However, changes can be aggregated so only a few changes will be required each year, and unless the SCP change dramatically, the rate of addition will slow down with each new addition.

**Acceptance Criteria**
- Craft GitHub action that runs the scripts and sends the results to a predetermined email address (note: currently emails from GitHub actions addressed to gov.bc.ca domain are blocked and do not reach destination)
- Configure OIDC by crafting roles and policies that allow to execute the task
- Document how to create the roles and policies and add instructions to the existing documents on how to use the script when executed using GitHub actions.
- If all is successful and works as expected. create tickets so SysOpe can add the OIDC layer to the accounts that are currently probed and were not configured with OIDC (the _github-oidc_ layer)

**Comment by crochcunill** [comment]
To let GitHub actions to run the script has been considered too create security risks, so this ticket will be cancelled.
A new ticket will be created to investigate if it would make sense to run the script using a lambda function
